study-builder/studies/testboston/emails/account_created.en.html

```diff
@@ -5,9 +5,9 @@
   "please_consent": "<strong><ins>However, you are not enrolled in the study until you read the Informed Consent.</ins></strong> You must read the Informed Consent study document, and if you wish to enroll, provide consent to participate by typing your name into the form.",
   "to_continue": "If you would like to continue and you have not yet read the Informed Consent,",
   "button": "Please click here to return to TestBoston.org",
-  "change_mind": "You may change your mind and decide to leave the study at any time by calling the study team by phone or email:",
+  "contact": "If you have questions about the study at any time, you may also contact us by phone or email: 617-525-4220 or info@testboston.org",
```

Comment:
There is already phone and email in the `account_created.ht.html` file with HTML `<a>` tags. Please remove from substitution text here.